data/styles/clear/include/Icons.mapcss

```diff
@@ -1425,6 +1428,9 @@ area|z18-[amenity=place_of_worship][religion=jewish]
 node|z18-[amenity=place_of_worship][religion=taoist],
 area|z18-[amenity=place_of_worship][religion=taoist]
 {icon-image: taoist-m.svg;}
+node|z18-[amenity=place_of_worship][religion=christian][denomination=mormon],
+area|z18-[amenity=place_of_worship][religion=christian][denomination=mormon]
+{icon-image: mormon-m.svg;}
```

Comment:
the z18- block in redundant as we have a z15- already;
looks like all of this z18- rules are just redundant, should be safe to delete them

Reply:
Removed redundant block